jumpapp

```diff
@@ -379,6 +379,10 @@ get_active_windowid() {
 list_stacking_order() {
     local name ids
     read -r name ids < <(xprop -root ' $0+\n' _NET_CLIENT_LIST_STACKING)
+    if [[ "$ids" == 'not found.' ]] || [[ "$ids" =~ 'no such atom' ]]; then
```

Comment:
Thanks for testing this on multiple machines to catch the multiple messages. Another idea is instead of trying to match on specific error messages, we could invert the match and look for something that only appears in the success output (maybe `(WINDOW)` or `0x`?). I like what you have though, let's try it and see if anyone reports issues.

BTW I double checked to see if we could use the process return status code, but it looks like it's always `0` whether the property exists or not.